tests/models/test_detection.py

```diff
@@ -10,27 +10,28 @@ def _collate_fn(batch):
     return tuple(zip(*batch))
 
 
+@torch.no_grad()
 def test_fasterrcnn():
-    model = FasterRCNN()
+    model = FasterRCNN(pretrained=False, pretrained_backbone=False)
 
-    image = torch.rand(1, 3, 400, 400)
+    image = torch.rand(1, 3, 224, 224)
     model(image)
 
 
 def test_fasterrcnn_train(tmpdir):
-    model = FasterRCNN()
+    model = FasterRCNN(pretrained=False, pretrained_backbone=False)
 
     train_dl = DataLoader(DummyDetectionDataset(), collate_fn=_collate_fn)
     valid_dl = DataLoader(DummyDetectionDataset(), collate_fn=_collate_fn)
 
-    trainer = pl.Trainer(fast_dev_run=True, default_root_dir=tmpdir)
+    trainer = pl.Trainer(fast_dev_run=True, logger=False, checkpoint_callback=False, default_root_dir=tmpdir)
```

Comment:
I like your idea of being defensive, but lightning ensures to disable these options with `fast_dev_run=True` as stated in [the docs](https://pytorch-lightning.readthedocs.io/en/stable/debugging.html#fast-dev-run) so how about keeping this line as is? 
```suggestion
    trainer = pl.Trainer(fast_dev_run=True, default_root_dir=tmpdir)
```